cypress/e2e/default/revalidate.cy.ts

```diff
@@ -33,7 +33,7 @@ describe('On-demand revalidation', () => {
     cy.request({ url: '/api/revalidate/?select=5', failOnStatusCode: false }).then((res) => {
       expect(res.status).to.eq(500)
       expect(res.body).to.have.property('message')
-      expect(res.body.message).to.include('Invalid response 404')
+      expect(res.body.message).to.include('could not refresh content for path /getStaticProps/withRevalidate/3/, path is not handled by an odb')
```

Comment:
Yes, it originates from `api.netlify.com`, and is shown here because of https://github.com/netlify/next-runtime/blob/f9f726f5f83df1776a765b0ecf4ab6647b83ee43/packages/runtime/src/templates/server.ts#L67.